backend/app/services/rag/local_data_plane/indexing.py

```
@@ -0,0 +1,174 @@
+    document_ref: str,
+    *,
+    db: Session,
+    index_owner_user_id: int | None = None,
+) -> dict:
+    """Delete document chunks from the local storage backend."""
+    kb = (
+        db.query(Kind)
+        .filter(
+            Kind.id == knowledge_base_id,
+            Kind.kind == "KnowledgeBase",
+            Kind.is_active == True,
```

Ruff (0.15.7)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)